api/MediaStreamAudioSourceNode.json

```diff
@@ -122,10 +122,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
@a2sheppy difference between `false` and `null` in this context?

Reply:
`null` means "unknown" while `false` means "not supported".